Wiki/de/index.md

```diff
@@ -0,0 +1,215 @@
+[Digiconomist](https://digiconomist.net/bitcoin-energy-consumption).
+Diese Zahl wird zwar viel diskutiert, sie zeigt aber sicherlich, dass
+eine Menge Energie von Bitcoin's SHA-256 Algorithmus verbraucht wird.
+Diese Energie könnte für weitaus sinnvollere Kalkulationen verwendet werden.
+Gridcoin basiert auf Dem Protokoll von Bitcoin, sodass es effektiv mit
+normaler "Verbraucher-Hardware" geschürft werden kann.
+Es ist somit unwahrscheinlich, dass diese Hardware künftig durch spezialisierte
+ASIC Mining Chips ersetzt wird.
+
+Gridcoin Research basiert auf dem Proof-of-Stake Algorithmus, der deutlich
+energieeffizienter ist. Energieeffizienz wurde bereits bei der Veröffentlichung
+von Peercoin, der ersten Proof-of-Stake Coin, verkündet. Prof-of-Stake erlaubt es
+den Schürfern ihre Computerressourcen ganz der Wissenschaft zu widmen, anstatt
+sie zur Unterhaltung der Blockchain zu verschwenden.
+
+## Offizielle Seiten
+  <!-- TODO: [Volunteers, Roles & Privileges](Volunteers,-Roles,-&-Privileges "wikilink") -->
+  - [Website](https://gridcoin.us/)
+  - [BOINCstats](https://boincstats.com/en/stats/-1/team/detail/118094994/overview)
+  - [Netsoft](http://www.boinc.netsoft-online.com/e107_plugins/boinc/bp_home.php)
+  - [SETIBZH](https://statsbzh.boinc-af.org/team.php?name=R3JpZGNvaW4=)
+  - [Free-DC Stats](https://stats.free-dc.org/stats.php?page=teambycpid&team=Gridcoin)
+  - [Statseb.fr Forum Badges](https://signature.statseb.fr/index.py?action=team_badge&teamid=100454)
+  - [Project Whitelist and Blacklist](https://gridcoin.ddns.net/pages/project-list.php)
+
+<!-- end list -->
+
+  - Block Explorer:
+      - [Gridcoinstats](https://gridcoinstats.eu/)
+      - [Gridcoin.network](https://gridcoin.network/)
+
+<!-- end list -->
+
+  - Quellcode:
+      - [Gridcoin-Research](https://github.com/gridcoin-community/Gridcoin-Research)
+      - [Gridcoin-Classic (**retired**)](https://github.com/gridcoin/Gridcoin-master)
+
+<!-- end list -->
+
+  - Weitere Quellen:
+      - [Gridcoin's GitHub community organization](https://github.com/gridcoin-community)
+          - [Website Development](https://github.com/gridcoin-community/Gridcoin-Site)
+          - [Gridcoin Tasks](https://github.com/gridcoin-community/Gridcoin-Tasks)
+          - [Marketing images](https://github.com/gridcoin-community/Gridcoin-Marketing)
+          - [Fan Site (Gridcoin Science)](https://gridcoin.science)
+
+<!-- end list -->
+
+  - Downloads:
+      - [Github](https://github.com/gridcoin-community/Gridcoin-Research/releases)
+      - [Launchpad](https://launchpad.net/gridcoin)
+      - [AUR Qt](https://aur.archlinux.org/packages/gridcoinresearch-qt)
+      - [AUR daemon](https://aur.archlinux.org/packages/gridcoinresearchd/)
+
+<!-- end list -->
+
+  - Handelsplätze:
+      - [Txbit](https://txbit.io/Trade/GRC/BTC)
+      - [SouthXchange](https://www.southxchange.com/Market/Book/GRC/BTC)
+
+<!-- end list -->
+
+  - Weitere Trading Möglichkeiten:
```

Comment:
```suggestion
  - Weitere Möglichkeiten zum Tauschen von GRC:
```

Reply:
I'd write "Handelnd" instead of tauschen.

"Weitere Möglichkeiten zum Handeln von GRC:"

Reply:
```suggestion
  - Weitere Möglichkeiten zum Handeln von GRC:
```